src/main/java/com/onaio/steps/adapters/ParticipantAdapter.java

```diff
@@ -68,22 +67,19 @@ public View getView(int position, View convertView, ViewGroup parent) {
 
     private void setTextInView(View view, Participant participantAtPosition) {
         TextView participantPidView = (TextView) view.findViewById(R.id.main_text);
-        ImageView cloudIcon = (ImageView) view.findViewById(R.id.cloud_icon);
         TextView createdAtView = (TextView) view.findViewById(R.id.sub_text);
         ImageView image = (ImageView) view.findViewById(R.id.main_image);
         image.setImageResource(getImage(participantAtPosition));
 
         participantPidView.setTextColor(Color.BLACK);
         String householdRow = participantAtPosition.getFormattedName()+" ("+context.getString(R.string.pid) + participantAtPosition.getParticipantID()+")";
         participantPidView.setText(householdRow);
-        cloudIcon.setVisibility(InterviewStatus.SUBMITTED.equals(participantAtPosition.getStatus()) ? View.VISIBLE : View.GONE);
```

Comment:
i have added another column in the household table for checking the `uploaded/submitted` status

Reply:
OK, I understand that this is the participant adapter and not the household adapter. Right?

Reply:
yes right.

Reply:
Sorry, I guess what I mean is that you talked about adding a column in the household table while this change affects the participant table instead. You didn't answer my initial question here https://github.com/onaio/steps-app/pull/137#discussion_r792355976

Reply:
I think we don't need to add a column in the participant table because we uploading household + member data only we are not uploading any participant record on the server.

Reply:
participant records are only sent via ODK app.

Reply:
So we lose the ability to show submitted participant surveys?

Reply:
Cool